CHANGELOG.md

```diff
@@ -1,3 +1,106 @@
+<a name="0.11.1"></a>
+## 0.11.1 (2022-05-05)
```

Comment:
Should we update this date again?

Reply:
It's really about the date of when this release was tagged so I think it should stay as is

Reply:
Ah, right 🤦🏻